partner/partner-management-service/src/main/java/io/mosip/pms/oidc/client/controller/ClientManagementController.java

```diff
@@ -22,29 +26,49 @@ public class ClientManagementController {
 	@Autowired
 	ClientManagementService clientManagementService;
 
+	@Autowired
+	AuditUtil auditUtil;
+
+	String msg = "mosip.clientmanagement.clients.retrieve";
+
+	String version = "1.0";
+
 	@RequestMapping(value = "/oidc/client", method = RequestMethod.POST, consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE)
-	public ResponseWrapper<ClientDetailResponse> createClient(
+	public ResponseEntity<ResponseWrapper<ClientDetailResponse>> createClient(
```

Comment:
why you modified the return type?

Reply:
I need to send a response back to the client after processing a request that's why

Reply:
Changed the return type